src/box/sql/sqlInt.h

```diff
@@ -3564,7 +3564,7 @@ sql_trigger_delete_step(struct Token *table_name, struct Expr *where);
  */
 uint64_t
 sql_trigger_colmask(Parse *parser, struct sql_trigger *trigger,
-		    ExprList *changes_list, int new, int tr_tm,
+		    ExprList *changes_list, int ref_mask, int tr_tm,
```

Comment:
Wouldn't it be better to replace `int new` with `bool is_new` or `bool is_old`? This way we can be sure that it only has two values, and we can choose an index for `prg->column_mask` depending on this argument value.

Reply:
Yes, `is_new` or `is_old` would be better and clear. According to comment above, 1 is passed for new mask and 0 for old mask, so I chose variable name `is_new`.
I agree, converting argument to boolean would be good too. Did it.
I'm not sure is it worth to convert passed o and 1 in callers to boolean explicitly because this conversion will be done implicitly and no sense to touch code there. Let me know your opinion.

Reply:
I think implicit casting `true` to 1 and `false` to 0 is fine for C.